doc/source/examples/cd_mmd_cifar10.ipynb

```diff
@@ -765,11 +765,8 @@
   }
  ],
  "metadata": {
-  "interpreter": {
-   "hash": "ffba93b5284319fb7a107c8eacae647f441487dcc7e0323a4c0d3feb66ea8c5e"
-  },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.8.13 ('detect_cpu_py38')",
```

Comment:
Please revert changes to this notebook as no content has changed (note that in it's current form it will throw a warning for any user that doesn't have an environment named `detect_cpu_py38`.

Reply:
Apologies, staged this one by mistake, now fixed.